### PR TITLE
Search smoke tests

### PR DESCRIPTION
**Pull request by barbaravaldez**
Add simple tests for the Notebooks View test.

### CODE REVIEW

**Comment by lucyzhang929** [comment]
Should we keep NotebookView tests separate from Notebook tests (notebookViews.test.ts and notebookViews.ts) since we'll probably be adding more tests specific to the view?

**Comment by barbaravaldez** [comment]
> Should we keep NotebookView tests separate from Notebook tests (notebookViews.test.ts and notebookViews.ts) since we'll probably be adding more tests specific to the view?

I think that makes sense. I'll make the change 😄

File: test/smoke/src/sql/areas/notebook/notebook.test.ts [hunk 1]
```diff
@@ -55,5 +56,21 @@ export function setup() {
...
+			assert(results.children !== undefined && results.children.length === 0);
+		});
+
+		it('Simple query search works correctly', async function () {
+			const app = this.app as Application;
+			// Adding a regex expression to not depend on specific results of files
+			const regexExpr = /[0-9]+( results in )[0-9]+( files - )/;
+			await app.workbench.quickaccess.runCommand('Notebooks: Focus on Search Results View');
+			const results = await app.workbench.sqlNotebook.view.searchInNotebook('hello');
+			assert(results.textContent !== '' && (results.textContent.match(regexExpr) || results.textContent.includes('No results found')));
```
Comment (Charles-Gagnon) [inline test/smoke/src/sql/areas/notebook/notebook.test.ts:73]: Why is "No results found" a valid result for this? Aren't you supposed to have found something?

File: test/smoke/src/sql/areas/notebook/notebookView.test.ts [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+			await app.workbench.quickaccess.runCommand('Notebooks: Focus on Search Results View');
+			const results = await app.workbench.sqlNotebook.view.searchInNotebook('');
+			assert(results.children !== undefined && results.children.length === 0);
+		});
+
+		it('Simple query search works correctly', async function () {
+			const app = this.app as Application;
+			// Adding a regex expression to not depend on specific results of files
+			const regexExpr = /[0-9]+( results in )[0-9]+( files)/;
+			await app.workbench.quickaccess.runCommand('Notebooks: Focus on Search Results View');
```
Comment (Charles-Gagnon) [inline test/smoke/src/sql/areas/notebook/notebookView.test.ts:23]: Might consider putting this in the NotebookView class as a helper function (`focus()`)

**Review by Charles-Gagnon — APPROVED** [review]

File: test/automation/src/sql/notebook.ts [hunk 3]
```diff
@@ -194,3 +196,25 @@ export class NotebookToolbar {
...
+	constructor(private code: Code) { }
+
+	async searchInNotebook(expr: string): Promise<IElement> {
+		await this.waitForSetSearchValue(expr);
+		await this.code.dispatchKeybinding('enter');
+		let selector = `${NotebookView.actualResult} `;
+		if (expr) {
+			selector += '.message span';
+		}
+		return await this.code.waitForElement(selector, undefined);
```
Comment (lucyzhang929) [inline test/automation/src/sql/notebook.ts:213]: If expr exists but doesn't return any results, will adding `'.message span'` throw an error here?

**Pull request merged**